packages/styles/scss/components/_accordion.scss

```diff
@@ -1,43 +1,44 @@
 @use "../tokens" as *;
 @use "../functions" as *;
 @import "../mixins";
+@import "../config";
 
-.ilo--accordion {
+.#{$prefix}--accordion {
+  // This provides a reference to the outerclass within nested declarations
+  $accordion: &;
```

Comment:
Use sensibly named references to refer to the outer class within nested declarations